package.json

```diff
@@ -114,6 +114,7 @@
     "gulp": "^4.0.0",
     "handlebars": "^4.0.10",
     "mocha": "^6.0.0",
+    "path-exists-cli": "^1.0.0",
```

Comment:
i hate that we need another dev dep just to do a `ls` but, sigh, cross platform

Reply:
Should be able to be `test -e npm-shrinkwrap.json && (...)`

Reply:
Would that work in windows @imurchie ?

Reply:
No idea. But given the usage of this, is it necessary to be cross-platform?

Reply:
@imurchie I'll change it to that since we only publish from Mac